tests/icons.test.js

```diff
@@ -25,8 +30,14 @@ icons.forEach((icon) => {
     expect(subject.source).toEqual(icon.source);
   });
 
-  test(`${icon.title} has an "svg" value`, () => {
+  test(`${icon.title} has a valid "svg" value`, () => {
     expect(typeof subject.svg).toBe('string');
+    const svgFileContents = fs
+      .readFileSync(svgPath, 'utf8')
+      .replace(/\r?\n/, '');
```

Comment:
Not sure about this, but would making the test asynchronous more efficient?

(didn't test if this suggestion works as is)

```suggestion
  test(`${icon.title} has a valid "svg" value`, async () => {
    expect(typeof subject.svg).toBe('string');
    const _svgFileContents = await fs.readFile(svgPath, 'utf8');
    const svgFileContents = _svgFileContents.replace(/\r?\n/, '');
```

Reply:
It doesn't changes performance